app/src/pages/admin/projectsPage/projects/projects.scss

```diff
@@ -0,0 +1,32 @@
+.toolbar {
+  padding: 10px 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.search {
+  width: 400px;
+}
+
+.buttons {
+  display: flex;
+}
+
+.toolbar-button {
+  margin-left: 10px;
```

Comment:
Accrording to the design, view toggling buttons shouldn't have space between them

Reply:
But on Dashboard, there is a space, probably should be the same here?
![image](https://user-images.githubusercontent.com/3634380/54438467-bb2c3f00-4747-11e9-9f71-ec8647c6a64e.png)

Reply:
I think it needs to be clarified with the UX team. Let's leave it as is for now, and if there shouldn't be space, fix it in another task